docs/status-and-issues.md

```diff
@@ -28,43 +28,47 @@ All of these configurations use Pulumi code within Python as the Infrastructure
-in active development and will result in this process being streamlined for these additional environments. Please follow
-[#80](https://github.com/nginxinc/kic-reference-architectures/issues/80) for details.
-4. We are currently using filebeat as our logging agent. This deployment requires that the correct paths to the container
-log directory are present in the deployment data. We have discovered that this differs based on the K8 provider. Please
-see [#76](https://github.com/nginxinc/kic-reference-architectures/issues/76) for more detail.
+
+1. The NGINX IC build/deploy process is currently under active development and support for IC will be standardized
+   across all providers. Follow [#81](https://github.com/nginxinc/kic-reference-architectures/issues/81) and
+   [#86](https://github.com/nginxinc/kic-reference-architectures/issues/86) for details. Currently, for all non-AWS
+   environments you have the option to specify either NGINX or NGINX Plus as your IC. The latter does require an active
+   subscription and a JWT to be included at build time. Please see the documentation for more details.
+2. The process via which the IP and FQDN are created and used is currently under active development, and will be
+   streamlined and standardized for all providers.
+   Follow [#82](https://github.com/nginxinc/kic-reference-architectures/issues/82) for details.
+3. The initial deployment was entirely built to work with AWS. As part of our reorganization the ability to use a
+   kubeconfig file was added, along with the necessary configuration to support additional standup options. This is
+   currently in active development and will result in this process being streamlined for these additional environments.
+   Please follow
+   [#80](https://github.com/nginxinc/kic-reference-architectures/issues/80) for details.
+4. We are currently using filebeat as our logging agent. This deployment requires that the correct paths to the
+   container log directory are present in the deployment data. We have discovered that this differs based on the K8
+   provider. Please see [#76](https://github.com/nginxinc/kic-reference-architectures/issues/76) for more detail.
 
 ## Known Issues / Caveats
+
 1. Currently, the use of the Elastic tooling has shown to be problematic under heavy load, with containers falling over
-and causing disruptions. Please see the tuning variables in the configuration file to adjust the number of replicas 
-deployed for the Elastic logstore to tune to your environment.
-2. The default Helm timeout is 5 minutes, which is acceptable for most managed clouds but tends to be too short for 
-single-vm or workstation deployments. Please see the configuration file variables to adjust the helm timeout as required
-for your environment.
+   and causing disruptions. Please see the tuning variables in the configuration file to adjust the number of replicas
```

Comment:
Is it possible to call out which specific configuration file is being referenced here?  Perhaps the intended audience would make this unnecessary but I wouldn't know instinctively where I need to look to tune configuration.

Reply:
Good point; fixed.